src/components/Status/Status.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import cx from "classnames";
+
+import "./status.css";
+
+const Status = ({ className, theme, ...props }) => {
+  return (
+    <div
+      className={cx("status", className, {
+        "status--warning": theme === "warning",
```

Comment:
This can actually be refactored to a single line:

```
className={cx("status", `status--${theme}`, className)}
```

And then define a default theme in your prop destructuring above with `theme = 'default'`